src/reducers/pending-breakpoints.js

```diff
@@ -87,25 +70,23 @@ function update(
 }
 
 function addBreakpoint(state, action) {
-  if (action.status !== "done") {
+  if (action.breakpoint.hidden || action.status !== "done") {
```

Comment:
Slightly different condition? Does this fix a bug too?

Reply:
cleans up the code a bit